### PR TITLE
adds a cartesian force input 

### PR DESCRIPTION
**Pull request by RussTedrake**
and (slightly) generalizes / robustifies the implementation of rigid body forces.

now we could apply, e.g. a wind field, as a separate system which simply applies forces through this new input.

### CODE REVIEW

File: systems/plants/@RigidBodyManipulator/RigidBodyManipulator.m [hunk 1]
```diff
@@ -2140,31 +2148,26 @@ function checkDirty(obj)
 
     function fr = constructInputFrame(model)
       %inputparents is an array of the parent RigidBodies of each joint.
```
Comment (RussTedrake) [inline systems/plants/@RigidBodyManipulator/RigidBodyManipulator.m]: this is the most significant change that could impact existing code. 
pretty sure I got it correct, but extra eyes would be appreciated.

**Comment by RussTedrake** [comment]
@peteflorence -- fyi, I did this is a first step towards accomplishing #67

File: systems/plants/dev/RigidBodySpatialForce.m [hunk 2]
```diff
@@ -0,0 +1,53 @@
+classdef RigidBodySpatialForce < RigidBodyForceElement
+
```
Comment (RussTedrake) [inline systems/plants/dev/RigidBodySpatialForce.m]: notice that this is in dev.  didn't write the test case yet.  and would benefit from a conversation with @tkoolen about whether i'm using his best tools (and even getting the transforms correct :)

Comment (tkoolen) [inline systems/plants/dev/RigidBodySpatialForce.m]: This looks good to me.
Re:
```
+      % todo: compare this to RigidBodyThrust, which is comparitively much
+      % simpler.  Do I have a good way of computing the correct jacobian in
+      % one shot?
```
What you have is pretty much the fastest way, but a shorter version would be something like
```
[J, v_indices] = manip.geometricJacobian(kinsol, 0, obj.frame_id, obj.frame_id, false); % geometric Jacobian with base world, end effector body that frame is attached to, expressed in frame, in terms of v
B(v_indices, obj.input_num) = J';
```
(haven't tested or even run this code, but you get the idea)

Comment (tkoolen) [inline systems/plants/dev/RigidBodySpatialForce.m]: Actually, what would make that solution significantly better is that it doesn't access kinsol.J directly, which would not be available if the kinsol is obtained from mex.

**Comment by RussTedrake** [comment]
this one has been sitting open for too long.  we can revisit if it's still necessary with the new c++ interfaces.

**Pull request closed without merging**